app/controllers/tutorials_controller.rb

```diff
@@ -57,10 +57,14 @@ def update
   # DELETE /tutorials/1
   # DELETE /tutorials/1.json
   def destroy
-    @tutorial.destroy
-    respond_to do |format|
-      format.html { redirect_to tutorials_url, notice: 'Tutorial was successfully destroyed.' }
-      format.json { head :no_content }
+    # only users who have created the tutorial can delete or admins
+    @user = current_user
```

Comment:
currrent_user is a devise helper, [check it out](https://github.com/plataformatec/devise#controller-filters-and-helpers)